sdk/network/Azure.ResourceManager.Network/src/Generated/ApplicationGatewayPrivateEndpointConnection.cs

```diff
@@ -23,7 +23,7 @@ namespace Azure.ResourceManager.Network
     public partial class ApplicationGatewayPrivateEndpointConnection : ArmResource
     {
         private readonly ClientDiagnostics _clientDiagnostics;
-        private readonly ApplicationGatewayPrivateEndpointConnectionsRestOperations _restClient;
+        private readonly ApplicationGatewayPrivateEndpointConnectionsRestOperations _applicationGatewayPrivateEndpointConnectionsRestClient;
```

Comment:
Is there any reason use a long name for `rest client`? I think `_restClient` is constrained in the context.